doc/dune

```diff
@@ -49,9 +49,10 @@
  ; EJGA: note this should've been fixed in dune master as of 05/03/2021
  (deps
   (package coq-core)
-  (package coq-stdlib)
+  (package rocq-core)
   (source_tree sphinx)
   (source_tree tools/coqrst)
+  ../config/coq_config.py
   unreleased.rst
   (env_var SPHINXWARNOPT))
```

Comment:
Shouldn't `COQRST_EXTRA` be listed there as well?

Reply:
Why not